packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -280,9 +289,10 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
 
     return (
       <Host
+        aria-checked={this.selectionMode === "ancestors" ? toAriaBoolean(this.selected) : undefined}
```

Comment:
Just want to confirm aria-checked is ok instead of aria-selected. The treeitem role says either is ok.

> Either aria-selected or aria-checked can be used to indicate selection for treeitem elements. Some user interfaces indicate selection with aria-selected in single-select trees and with aria-checked in multi-select trees.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/treeitem_role

It seems like depending on the selectionMode it should use either aria-checked (multiple) or aria-selected (single)

Reply:
@aPreciado88 It looks like based on @driskull comment above, there may be a need for both depending on the `selectionMode` type, where only one would be present for the `selectionMode` value.